app/controllers/trainees/deferrals_controller.rb

```diff
@@ -5,9 +5,9 @@ class DeferralsController < ApplicationController
     before_action :authorize_trainee
 
     PARAM_CONVERSION = {
```

Comment:
Could we move `PARAM_CONVERSION` into `TodayTomorrowOtherDateForm`? This would remove the need to define it in each controller that has to do date conversion.

Reply:
nice idea! thanks 👍